functions-and-operators/string-functions.md

```diff
@@ -23,36 +23,18 @@ TiDB 支持使用大部分 MySQL 5.7 中提供的[字符串函数](https://dev.m
 >
 > `ASCII()` 只能处理那些用 8 个二进制数字（即单个字节）来表示的字符。
 
-示例：
+查询示例：
 
 ```sql
-SELECT ASCII('A');
-
-+------------+
-| ASCII('A') |
-+------------+
-|         65 |
-+------------+
+SELECT ASCII('A'), ASCII('TiDB'), ASCII(23);
 ```
 
-```sql
-SELECT ASCII('TiDB');
-
-+---------------+
-| ASCII('TiDB') |
-+---------------+
-|            84 |
-+---------------+
-```
+返回结果：
 
 ```sql
-SELECT ASCII(23);
-
-+-----------+
-| ASCII(23) |
-+-----------+
-|        50 |
-+-----------+
+ ASCII('A') | ASCII('TiDB') | ASCII(23) 
```

Comment:
Why not use the previous one?
+---------------+
| ASCII('TiDB') |
+---------------+
|            84 |
+---------------+

Reply:
Originally divided into 3 examples with minor differences, they were combined to avoid redundancy in the documentation, which would have otherwise required writing 3 query examples and 3 response examples.

Reply:
I mean why do you change the output format, the previous one is surrounded with '-', '+' and '|'. The current format seems a little strange

Reply:
When executing the  SQL, it returns as follows: The format does not match yours, which is correct?
![iShot_2024-01-15_17 37 22](https://github.com/pingcap/docs-cn/assets/92153708/08eb9e4d-c006-4979-a399-37455dfb40a6)

Reply:
I think it may be caused by using different client or client configs. In my env using mysql client, the output is with '-', '+' and '|'. And please refer to other tidb doc as reference:).